operator-framework-core/src/main/java/io/javaoperatorsdk/operator/processing/dependent/kubernetes/GenericKubernetesResourceMatcher.java

```diff
@@ -24,6 +25,9 @@ public class GenericKubernetesResourceMatcher<R extends HasMetadata, P extends H
   private static final String OP = "op";
   public static final String METADATA_LABELS = "/metadata/labels";
   public static final String METADATA_ANNOTATIONS = "/metadata/annotations";
+  // without knowing the CRD we cannot ignore status as it may not be a subresource, so if it's
```

Comment:
can you give an example when we cannot ignore status?

Reply:
(I mean by definition does not have to be a sub-resource but I guess that usually is just a bug, or are there any real life examples?

Reply:
The only reference I have for this was when there was a change in fabric8 a while ago that made the assumption we could treat status as a subresource, it was brought up then that some users had legacy crs with non-subresource status fields.

Reply:
I would threat those as exceptions. We can tell them, to make their custom marcher for that case.